org.mattbas.Glaxnimate.yml

```diff
@@ -0,0 +1,27 @@
+app-id: org.mattbas.Glaxnimate
+runtime: org.kde.Platform
+runtime-version: "6.3"
```

Comment:
the README only mention qt5. Are you sure you want Qt6?

Reply:
Yeah, works with both. I ported everything to work on Qt6 as of the latest release